### ISSUE TITLE
As a user I cannot use passbolt API in a subdirectory

### ISSUE THREAD

**Issue by stripthis** [passbolt/passbolt_api#252]
* Passbolt Version: 2.x
* Platform and Target:
-- Operating system: Any
-- PHP: 7.x
-- Web server: Apache / Nginx
-- Database server: Any

### What you did
See. https://community.passbolt.com/t/bad-urls-in-emails/640

### What happened
Url in emails is not set properly.

### What you expected to happen
I should be able to install passbolt with urls such as domain.tld/passbolt

**Comment by cmouse** [comment]
:+1: I am facing this issue too. I can install passbolt into domain.tld/passbolt, but I cannot set fullBaseUrl to `domain.tld/passbolt`, because then it will add `passbolt` twice to all URLs in browser. If I leave it out, then emails do not contain `/passbolt/` in the URL.

**Comment by farfade** [comment]
Hello,

In addition to fixing this for emails, please do not forget to fix it also for the "logout" button (it calls the base url + /auth/logout whereas it should call the base url + subdirectory + /auth/logout). Not sure why it works for other menus (passwords, users...) but not for "logout".

Regards,

Romain

**Comment by stripthis** [comment]
Hello, 

This issue should be fixed with version 2.8. In order to run passbolt from a sub directory you need to defined the following in passbolt.php:
```
return [
    'App' => [
       // A base URL to use for absolute links.
        // The fully qualified domain name (including protocol) to your application’s root
        // e.g. where the passbolt instance will be reachable to your end users.
        // This information is need to render images in emails for example.
        'fullBaseUrl' => 'https://www.passbolt.test',
        // OPTIONAL you can specify the base directory the app resides in
        // usefull for example if you are running passbolt in a subdirectory like localhost/passbolt
        // Ensure your string starts with a / and does NOT end with a /
        'base' => '/subdir'
    ],
    // rest of your config goes here
```
